lib/PhpParser/Node/Stmt/MatchArm.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace PhpParser\Node\Stmt;
+
+use PhpParser\Node;
+
+class MatchArm extends Node\Stmt
+{
+    /** @var null|Node\Expr[] */
+    public $condList;
+    /** @var Node\Expr */
+    public $body;
+
+    /**
+     * @param null|Node\Expr[] $condList
+     */
+    public function __construct($condList, Node\Expr $body, array $attributes = []) {
+        $this->condList = $condList;
+        $this->body = $body;
+
+        parent::__construct($attributes);
```

Comment:
```suggestion
        $this->attributes = $attributes;
```
We don't parent parent constructor for performance reasons.